packaging/nntrainer.spec

```diff
@@ -393,15 +396,19 @@ popd
 
 %if 0%{?gcov:1}
 mkdir -p gcov-obj
-find . -name '*.gcno' -exec cp '{}' gcov-obj ';'
+# the files under Applications & nnstreamer & test directories are not line coverage target.
+find . ! -path "*/gcov-obj/*" ! -path "*/Applications/*" ! -path "*/nnstreamer/*" ! -path "*/test/*" -name '*.gcno' ! -name "meson-generated*" ! -name "sanitycheck*" -exec cp --parents '{}' gcov-obj ';'
```

Comment:
gcno files contains `build` in the file path.
Change lcov option or directory to build before copying the gcno files.